alchemiscale_fah/compute/service.py

```diff
@@ -327,21 +329,24 @@ async def async_cycle(
                 async_tasks.remove(async_task)
 
             # attempt to claim a new task, add to execution
-            self.logger.info("Attempting to claim an additional task")
-            task_sks: List[ScopedKey] = self.client.claim_tasks(
-                scopes=self.scopes,
-                compute_service_id=self.compute_service_id,
-                count=self.claim_limit,
-                protocols=self.settings.protocols,
-            )
+            if self.claim_limit - len(async_tasks) > 0:
```

Comment:
```suggestion
            if tasks_to_claim := self.claim_limit - len(async_tasks):
```

Reply:
I explicitly want this to be greater than 0; though it shouldn't be possible for the service to claim more tasks than `claim_limit`, we want this to be guardrail.